robottelo/performance/stat.py

```diff
@@ -6,6 +6,8 @@
 def generate_stat_for_concurrent_thread(thread_name, time_list,
                                         stat_file_name, bucket_size,
                                         num_buckets):
+    if bucket_size == 0:
```

Comment:
Yes! That's for purpose of here: https://github.com/danuzclaudes/robottelo/blob/perf-constants/robottelo/test.py#L455,  where if a thread/client would conduct only 9 iterations, and given `num_buckets` to be 10, then the bucket-size would be 9/10 = 0..thus causing `ZeroDivisionError`. 

Normally, if we conduct 5k iterations in total, and test by 10 threads, each thread would be evenly distributed by 500 iterations; we further group these 500 time points by 10 buckets in order to compute stat and generate charts. In this case, the bucket-size would just be 500/10 without problem. And this is the logic for `_write_stat_per_client_bucketized` here: https://github.com/danuzclaudes/robottelo/blob/perf-constants/robottelo/test.py#L538